client/my-sites/plugins-wpcom/style.scss

```diff
@@ -197,3 +199,129 @@
 		margin: 0 auto;
 	}
 }
+
+.plugins-wpcom__jetpack-plugin-item.card {
+	display: flex;
+	flex-direction: row;
+	padding: 0;
+
+	.plugins-wpcom__plugin-link {
+		display: block;
+		flex-grow: 1;
+		font-size: 14px;
+		padding: 16px;
+		overflow: hidden;
+	}
+
+	.plugins-wpcom__plugin-name {
+		color: $gray-dark;
+		display: block;
+		line-height: 21px;
+		overflow: hidden;
+		text-align: left;
+		text-overflow: ellipsis;
```

Comment:
Should we avoid `ellipsis` here, since it will crop the Jetpack plugin title on some widths (try with `700px` and `500px`). 

![jetpackcrop](https://cloud.githubusercontent.com/assets/1182160/20410129/4a2e4e3e-ad24-11e6-971d-6ee2c8b484a7.png)